docs/docs/_index.md

```diff
@@ -12,7 +12,56 @@ Hopefully you will find everything you need in here, if not feel free to open an
 
 ## Examples
 
-Just like in v4 we have a number of examples to look at.
+Just like in v4 we have a number of [examples](examples/) to look at.
+
+## Content Cache Duration
+
+Grafana don't have any webhook or similar ways of giving information to the operator that a grafana resource like a dashboard have been changed.
+Due to this the Grafana operator constantly have to check with the Grafana API and see if something have changed in the dashboard.
+
+That is why we introduced `spec.contentCacheDuration`, this is a configuration that makes it possible to tell the operator
+how often should it check with the Grafana instance if the dashboard matches the the settings that is defined in Kubernetes.
+
+So if for example a dashboard is changed the operator will come in a overwrite those settings after `24h` by default.
+If you never want the operator to check if the dashboard have changed you need to set this value to `0h`.
```

Comment:
Probably blind but i can't find any definition of this setting be 24h.
But I do find a value of 5m https://github.com/grafana-operator/grafana-operator/blob/2459f1a54f1f0c8d9d7c76fae66ae5f762bf8ada/api/v1beta1/grafanadashboard_types.go#L37

I don't remember what settings we have discussed, but all I know is what the docs says. We need to verify this.